README.md

```diff
@@ -1,35 +1,46 @@
+Example 2 - I would like to make a HTTP request to the [TFL API](https://api.tfl.gov.uk/), to find out which buses go between London Bridge and Finsbury Park.
 
-Example 2 - I would like to use the Request module to make a HTTP request to the [TFL API](https://api.tfl.gov.uk/), to find out which busses go between London Bridge and Finsbury Park.
+### The challenge
 
-### Your challenge
-Create a function called **myRequest** that has similar set of features to the Node Request module.
+Create a function called `myRequest` which makes a request to an API url using an http GET request.
 
-The npm Request module simply creates a wrapper around the Node core http module. Your task it to create your own request function `myRequest` that performs in the same way as the Request npm module, which is used in the app.js file in this repo.
+It should take two arguments: the url to make the request to, and a callback function.
+
+The callback function will have two arguments:
+
+  1. error (String: if an error occurred)
+  2. response (Object: includes the body & statusCode of the request)
 
 **Hint**: relate to your knowledge of streams and sending data in chunks.
 
 To complete this challenge you will need to use the [`http.get`](https://nodejs.org/api/http.html#http_http_get_options_callback) method of the node core `http` module.
 
 ### Start here!
 1) Clone this repo & run `npm install`
-2) Run the app.js file in your command line with the command `node app.js`. This is the response from the request module that you're trying to recreate with your myRequest function. There is no need to change the helper function code.
-3) Open app.js and follow the instructions to create your own request function. 
+2) Run `npm test` in your command line to start the test watcher. The tests should fail as your function doesn't exist yet
+3) Open `app.js` and write your function!
+
+### Tests
+There are two test sets in `app.test.js` (the second is commented out). The first test will run three assertions: 
+1) Checks there are no errors
+2) Checks that the API response passed to the callback has a status code of 200
+3) Checks that the API response passed to the callback has the correct json
+
+The second, commented-out test will check your function works with https requests (bonus round, see below...). Uncomment it when you're ready to test in the bonus round!
 
 ### Bonus round
-Can you alter your myRequest function to check which protocol the url is using (http or https) and make the suitable request. You will need to require in the `https` node core module. You can test this by using the `https://jsonplaceholder.typicode.com/users/1` url.
+You can alter your myRequest function to check which protocol the url is using (http or https) and make the suitable request. You will need to require in the `https` Node core module. You can test this by uncommenting the second test in `app.test.js`.
```

Comment:
Maybe add an intro to this asking them to try making a request to the Pokeapi (or anything that uses https) so they can see it fail

Reply:
I've added a more detailed intro, how's [this](https://github.com/foundersandcoders/mc-request-module-workshop/pull/8/commits/861b2debb769660884c075ecf50a3078c1daa126)?